eng/pipelines/coreclr/pr.yml

```diff
@@ -30,176 +12,111 @@ jobs:
 - template: /eng/pipelines/common/checkout-job.yml
 
 #
-# Debug builds
-#
-- template: /eng/pipelines/common/platform-matrix.yml
-  parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
-    buildConfig: debug
-    platforms:
-    - Windows_NT_x64
-    - Windows_NT_x86
-    jobParameters:
-      testGroup: innerloop
-
-#
-# Checked builds
+# Build CoreCLR
 #
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
     jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
     buildConfig: checked
     platforms:
-    - Linux_arm
-    - Linux_arm64
-    - Linux_musl_x64
-    - Linux_x64
     - OSX_x64
-    - Windows_NT_arm
-    - Windows_NT_arm64
-    - Windows_NT_x64
+    - Linux_x64
     - Windows_NT_x86
-    jobParameters:
-      testGroup: innerloop
-
-#
-# Release builds
-#
-- template: /eng/pipelines/common/platform-matrix.yml
-  parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
-    buildConfig: release
-    platforms:
-    - Linux_arm64
-    - Linux_musl_x64
-    - Linux_rhel6_x64
-    - OSX_x64
-    - Windows_NT_arm
-    - Windows_NT_arm64
     - Windows_NT_x64
+    - Windows_NT_arm64
     jobParameters:
       testGroup: innerloop
 
 #
-# Checked test builds
+# Build CoreFX using live CoreLib from CoreCLR
 #
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/build-test-job.yml
+    jobTemplate: /eng/pipelines/libraries/build-job.yml
     buildConfig: checked
     platforms:
-    - Linux_arm
-    - Linux_arm64
     - OSX_x64
-    - Windows_NT_arm
-    - Windows_NT_arm64
-    - Windows_NT_x64
+    - Linux_x64
     - Windows_NT_x86
+    - Windows_NT_x64
+    - Windows_NT_arm64
     jobParameters:
-      testGroup: innerloop
+      isOfficialBuild: false
+      testScope: empty
+      useLiveCoreClr: true
 
 #
-# Checked JIT test executions
+# Test builds using live CoreFX
```

Comment:
Past this point we can probably remove everything. The test builds should be done by `coreclr/pr.yml` and `libraries/pr.yml`

Reply:
As discussed in today meeting, my current assumption is that I'll rename this to a new pipeline (perhaps eng/pipelines/root.yml as I think you yourself suggested but I'll gladly listen to other suggestions) and for now I'll put back the original CoreCLR PR pipeline as is. With this modification, I think it makes sense to keep the CoreFX test logic in the new pipeline as it's something I've already fixed and can be reused (or rather kept and conditionally filtered out according to our plan from earlier today). Please let me know if you think otherwise.